cli/src/test/kotlin/de/deutschepost/sdm/cdlib/archive/ArchiveCommandIntegrationTest.kt

```
@@ -173,7 +174,8 @@ class ArchiveCommandIntegrationTest(@Value("\${artifactory-azure-identity-token}
         val repository = artifactory.repository(repoName)
         try {
             repository.delete(releaseName_build)
-        } catch (_: Exception) {
+        } catch (e: Exception) {
+            logger.error("Exception caught while attempting to delete the releaseName_build folder", e)
```

The code fix involves replacing an empty catch block with a logging statement. The logging statement will capture any exceptions caught instead of silently ignoring them, as per the fix instructions. This way, any issue during the deletion of the repository folder will be logged for further investigation.<br><br>[Related Sonar Issue](http://localhost:9000/issues?open=15156638-4782-4aec-a774-3f852ff7c2ca)<br>[Related Sonar Rule](http://localhost:9000/coding_rules?open=kotlin:S108&rule_key=kotlin:S108)